### ISSUE TITLE
Cannot import a GraphQL API

### ISSUE THREAD

**Issue by wasuradananjith** [wso2/product-apim-tooling#708]
**Description:**
An error occurs when importing a GraphQL API.

**Affected Product Version:**
APIM 4.0.0-RC1

**Steps to reproduce:**
1. Create a GraphQL API using the Publisher portal according to [1].
2. Export the created API using apictl.
3. Try to import it again to another environment and you will get the below error.
```
wasura@wasura:~/Desktop/test$ apictl import api -f /home/wasura/.wso2apictl/exported/apis/dev/Test_1.0.0.zip -e dev
Error importing API.
Status: 500 
Response: {"code":500,"message":"Internal server error","description":"The server encountered an internal error. Please contact administrator.","moreInfo":"","error":[]}
apictl: Error importing API Reason: 500 
Exit status 1
```
4. Server logs will look like below.
```
[2021-04-22 08:02:14,412] ERROR - GlobalThrowableMapper An unknown exception has been captured by the global exception mapper.
java.lang.NullPointerException: null
	at org.wso2.carbon.apimgt.rest.api.publisher.v1.common.mappings.ImportUtils.importApi(ImportUtils.java:284) ~[org.wso2.carbon.apimgt.rest.api.publisher.v1.common_9.0.174.jar:?]
	at org.wso2.carbon.apimgt.rest.api.publisher.v1.common.ImportExportAPIServiceImpl.importAPI(ImportExportAPIServiceImpl.java:191) ~[?:?]
	at org.wso2.carbon.apimgt.rest.api.publisher.v1.impl.ApisApiServiceImpl.importAPI(ApisApiServiceImpl.java:3905) ~[classes/:?]
	at org.wso2.carbon.apimgt.rest.api.publisher.v1.ApisApi.importAPI(ApisApi.java:1192) ~[classes/:?]
	at sun.reflect.GeneratedMethodAccessor1120.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_251]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_251]
```

[1] https://apim.docs.wso2.com/en/4.0.0/design/create-api/create-a-graphql-api/

**Comment by pittavini** [comment]
I still see this issue, using 
APIM 4.0.0
and 
apictl  ./apictl version
Version: 4.1.0-m3
Build Date: 2022-01-07 11:40:03 UTC
apictl  ./apictl import api -f /tmp/Test_1.0.0.zip -e dev
Error importing API.
Status: 500
Response: {"code":500,"message":"Internal server error","description":"The server encountered an internal error. Please contact administrator.","moreInfo":"","error":[]}
apictl: Error importing API Reason: 500
Exit status 1

java.lang.NullPointerException: null
        at org.wso2.carbon.apimgt.rest.api.publisher.v1.common.mappings.ImportUtils.importApi(ImportUtils.java:284) ~[org.wso2.carbon.apimgt.rest.api.publisher.v1.common_9.0.174.jar:?]
        at org.wso2.carbon.apimgt.rest.api.publisher.v1.common.ImportExportAPIServiceImpl.importAPI(ImportExportAPIServiceImpl.java:191) ~[?:?]
        at org.wso2.carbon.apimgt.rest.api.publisher.v1.impl.ApisApiServiceImpl.importAPI(ApisApiServiceImpl.java:3905) ~[classes/:?]
        at org.wso2.carbon.apimgt.rest.api.publisher.v1.ApisApi.importAPI(ApisApi.java:1192) ~[classes/:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]

**Comment by wasuradananjith** [comment]
Hi @pittavini,
This issue is already fixed for 4.0.0 via a U2 update and is not reproducible. If you have a WSO2 subscription make sure to update [1] your API 4.0.0 pack to the latest and try with apictl 4.0.1 [2].

Otherwise, you can download APIM 4.1.0-M3 from here [3] and use apictl 4.1.0-M3 [4] with it.

[1] https://updates.docs.wso2.com/en/latest/updates/update-tool/
[2] https://github.com/wso2/product-apim-tooling/releases/tag/v4.0.1
[3] https://github.com/wso2/product-apim/releases/tag/v4.1.0-m3
[4] https://github.com/wso2/product-apim-tooling/releases/tag/v4.1.0-m3
